x-pack/test/api_integration/config.ts

```diff
@@ -34,6 +34,7 @@ export async function getApiIntegrationConfig({ readConfigFile }: FtrConfigProvi
         '--xpack.data_enhanced.search.sessions.enabled=true', // enable WIP send to background UI
         '--xpack.data_enhanced.search.sessions.notTouchedTimeout=15s', // shorten notTouchedTimeout for quicker testing
         '--xpack.data_enhanced.search.sessions.trackingInterval=5s', // shorten trackingInterval for quicker testing
+        '--xpack.data_enhanced.search.sessions.cleanupInterval=5s', // shorten cleanupInterval for quicker testing
```

Comment:
FYI: AFAIK, If the new tests rely upon these custom configs, then they will most likely fail in cloud test since they use a different config. I am not sure though where is this config and how to update it. cc @liza-mae